python/sdk/tests/test_iam.py

```diff
@@ -95,11 +95,11 @@ def test_create_user(self, unwrap):
             dict(
                 handle=self.second_user,
                 permission=Permission.SERVICE.value,
-                # name='Rob',
+                name='Rob',
```

Comment:
just curious why this was commented out before

Reply:
i'm sure it got through on accident. sometimes i get paranoid and think that a PASS means that I'm not actually comparing the right objects and i comment things out to force a fail lol